packages/actor-extract-links-extract-tree/lib/ActorExtractLinksTree.ts

```diff
@@ -0,0 +1,74 @@
+import type {
+  IActionExtractLinks,
+  IActorExtractLinksOutput, IActorExtractLinksArgs,
+} from '@comunica/bus-extract-links';
+import { ActorExtractLinks } from '@comunica/bus-extract-links';
+import type { ILink } from '@comunica/bus-rdf-resolve-hypermedia-links';
+import type { IActorTest } from '@comunica/core';
+import { DataFactory } from 'rdf-data-factory';
+import type * as RDF from 'rdf-js';
+
+const DF = new DataFactory<RDF.BaseQuad>();
+
+/**
+ * A comunica Extract Links Tree Extract Links Actor.
+ */
+export class ActorExtractLinksTree extends ActorExtractLinks {
+  public static aNodeType = DF.namedNode('https://w3id.org/tree#node');
+  public static aRelation = DF.namedNode('https://w3id.org/tree#relation');
+  private static readonly rdfTypeNode = DF.namedNode('http://www.w3.org/1999/02/22-rdf-syntax-ns#type');
```

Comment:
Should they be variable of the actor

Reply:
Yeah, it's fine to keep them as static vars in the class.

Reply:
> Yeah, it's fine to keep them as static vars in the class.

I mean variable from the config, like https://github.com/comunica/comunica-feature-link-traversal/blob/master/engines/config-query-sparql-link-traversal/config/extract-links/actors/predicates-common.json with the `predicateRegexes`

Reply:
Ah, like that.

No, I don't think this is needed at this stage.
I suspect these IRIs will not require frequent changes.